src/wikitree/wikitree.js

```diff
@@ -0,0 +1,16 @@
+import getData from "../axios/getData";
+
+export default async function getWikitreeComImages(wikitreeId) {
```

Comment:
I would call this function `getWikitreeImageUrl`, and rename the file accordingly

Reply:
Ok